### DISCUSSION TITLE
Custom App Registration & Secret Authentication

### DISCUSSION THREAD

**Discussion by jrollmann** [Micke-K/IntuneManagement#262] [Q&A]
Is there a way to use a secret and access the UI for this utility or is it only possible for silent jobs?

**Comment by Micke-K** [comment]
So sorry, I completely missed this.

Yes, you can login with secret in the UI.

Go to Settings and fill in the App Id and Secret and check "Login with App ID in UI"

![image](https://github.com/user-attachments/assets/f3a9cb5f-35dd-4c28-8e90-d6671ca02a52)

Cheers!
